packages/google-pay-integration/src/google-pay-bigcommerce-payments/create-google-pay-bigcommerce-payments-button-strategy.ts

```diff
@@ -0,0 +1,36 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { getScriptLoader } from '@bigcommerce/script-loader';
+
+import { PayPalSdkHelper } from '@bigcommerce/checkout-sdk/bigcommerce-payments-utils';
+import {
+    CheckoutButtonStrategyFactory,
+    toResolvableModule,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import createGooglePayScriptLoader from '../factories/create-google-pay-script-loader';
+import GooglePayButtonStrategy from '../google-pay-button-strategy';
+import GooglePayPaymentProcessor from '../google-pay-payment-processor';
+
+import GooglePayBigCommercePaymentsGateway from './google-pay-bigcommerce-payments-gateway';
+
+const createGooglePayBigCommercePaymentsButtonStrategy: CheckoutButtonStrategyFactory<
+    GooglePayButtonStrategy
+> = (paymentIntegrationService) => {
+    return new GooglePayButtonStrategy(
```

Comment:
Do we need to implement **Button** and **Customer** strategies in scope of the new bigcommerce payment provider?

Reply:
Yes, we should

Reply:
What about buy now flow? Do we need it for this provider?

Reply:
Yep